frontend/src/hooks/useLogin.ts

```diff
@@ -31,9 +31,19 @@ export const useLogin: LoginHook = () => {
       setResult({ ...result, isLoading: false });
       const { data, error } = await client.auth.signInWithPassword(credentials);
       setResult({ ...result, data, error });
+      if (data && !error) {
+        window.location.reload();
```

Comment:
What's the reload for here?

Reply:
When the Login action is done and there is no error, it will reload pages and show the buttons. If not reload, then you have to refresh to update the navbar buttons

Reply:
Try changing line 45 ish in AuthModel.tsx, it shouldn't be `!!error` rather it should be `!error`, as we should only perform the submit function when error is nil.

Reply:
It's better to leave one overall, generic purpose in hooks. Something like this is reactive to what the hook does, better handled in externally rather than within a hook.

Reply:
Yeah I think so, I will try after a successful click then reload inside the click event